storage/rocksdb/ha_rocksdb.cc

```diff
@@ -8381,12 +8381,23 @@ ha_rocksdb::generate_cf_name(const uint index, const TABLE *const table_arg,
     return "";
   }
 
-  // If we didn't find any partitioned/non-partitioned qualifiers, return the
-  // comment itself.  NOTE: this currently handles returning the cf name
-  // specified in the index comment in the case of no partitions, which doesn't
-  // use any qualifiers at the moment. (aka its a special case)
-  if (cf_name.empty() && !key_comment.empty()) {
-    return key_comment;
+  // If we didn't find any partitioned/non-partitioned qualifiers, DO NOT return
+  // the  comment itself. FB Behavior is exactly the opposite which then causes
```

Comment:
(since you have to repush because of a comment below, if not, ignore)
- double space between "the" and "comment"
- please narrow FB version to 5.6, they could change this in 8.0